Packs/FiltersAndTransformers/Scripts/ConcatFormat/README.md

```diff
@@ -1,187 +1,87 @@
+| ctx_inc | \`demisto\` context: Input 'incident' \(no quotation\) on \`From previous tasks\` to enable $\{incident.\} expression in DT. |
+| variable_markers | The pair of start and end markers to bracket a variable name |
+| keep_symbol_to_null | Set to true not to replace a value if the variable is null, otherwise false. |
 
 ## Outputs
 ---
 There are no outputs for this script.
 
 ## Getting Started
 ---
-The script builds a text from a template text which includes variables such as:
-This is a test message for ${user_name}.
-
-The template will be formatted to `This is a test message for John Doe.` by replacing variable parameters.
-
-By default, a variable name starts with `${` and ends with `}` . You can change the start marker and end marker by specifying the `variable_markers` parameter.
+The transformer concatenates prefix and suffix which supports DT expressions to the string.
 
 ## Examples
 ---
 
-### Replace variables in a text based on the context data.
-
-#### Command
-```
-!SetWithTemplate key=out template=${lists.Template}
-```
-
-#### Lists Library
-Template:
-```
-My name is ${first_name} ${last_name}.
-```
-
-#### Context Data
-```
-{
-  "first_name": "John",
-  "last_name": "Doe"
-}
-```
-
-#### Output
-```
-{
-  "out": "My name is John Doe."
-}
-```
-
----
-
-### Change the variable start and end marker to the windows command shell style such as %name%.
-
-#### Command
-```
-!SetWithTemplate key=out template=${lists.Template} variable_markers=%,%
-```
-
-#### Lists Library
-Template:
-```
-My name is %first_name% %last_name%.
-```
-
-#### Context Data
-```
-{
-  "first_name": "John",
-  "last_name": "Doe"
-}
-```
-
-#### Output
-```
-{
-  "out": "My name is John Doe."
-}
-```
-
----
-
-### Change the variable start and end marker to the UNIX shell style such as $name.
+### Build an email address from a user ID by appending a domain
 
-#### Command
-```
-!SetWithTemplate key=out template=${lists.Template} variable_markers=$
-```
-
-#### Lists Library
-Template:
-```
-My name is $first_name $last_name.
-```
+#### Parameters
+| **Argument Name** | **Value** | **Note** |
+| --- | --- | --- |
+| value | jdoe | |
+| prefix | | |
+| suffix | @${domain} | |
+| ctx_data | . | Make sure that **From previous tasks** is selected |
+| ctx_inputs | | |
+| ctx_inc | | |
+| variable_markers | | |
+| keep_symbol_to_null | | |
 
 #### Context Data
 ```
 {
-  "first_name": "John",
-  "last_name": "Doe"
+  "domain": "paloaltonetworks.com"
 }
 ```
 
 #### Output
 ```
-{
-  "out": "My name is John Doe."
-}
+jdoe@paloaltonetworks.com
 ```
 
----
-
-### Keep variable names if they are missing in the context.
-
-#### Command
-```
-!SetWithTemplate key=out template=${lists.Template} keep_symbol_to_null=true
-```
+### Build an email address by adding a user ID to the domain
 
-#### Lists Library
-Template:
-```
-My name is ${first_name} ${last_name}.
-```
+#### Parameters
+| **Argument Name** | **Value** | **Note** |
```

Comment:
Can you please add the missing fields ? There are no value or note for some of the fields